rotkehlchen/chain/bitcoin/bch/manager.py

```diff
@@ -100,18 +118,39 @@ def _get_haskoin_address(
 
         return None
 
+    def _check_haskoin_health(self, base_url: str) -> None:
+        """Check the health of a haskoin-store API.
+        Raises RemoteError if the health check fails.
```

Comment:
```suggestion
        Raises RemoteError if the health check fails. This is done like this over a boolean return
        since it's already used exclusively in functions that may already raise RemoteError
```